### PR TITLE
Add some logging when skipping the streaming of the log message.

### PR DESCRIPTION
**Pull request by benguillet**

### CODE REVIEW

File: kinesis.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ func (a *KinesisAdapter) Stream(logstream chan *router.Message) {
 		}
 
 		if streamName == "" {
+			debugLog("The stream name is empty, couldn't match the template. Skipping the log.\n")
```
Comment (phobologic) [inline kinesis.go:58]: Should we log this on startup also if things are broken?

Comment (benguillet) [inline kinesis.go:58]: Not sure what you mean exactly by things are broken? If the template cannot compile we will return the error: https://github.com/remind101/logspout-kinesis/blob/master/kinesis.go#L31

Comment (phobologic) [inline kinesis.go:58]: Actually, yeah, this with your other commit makes more sense- provided we deal w/ the typo situation I metnioned over there.

**Comment by ejholmes** [comment]
:+1:

File: kinesis.go (hunk 1 above)
Comment (benguillet) [inline kinesis.go:58]: I have no way to check for a typo, unless: 1) the template compilation crash, if which case you'll get alerted: https://github.com/remind101/logspout-kinesis/blob/master/kinesis.go#L31, or 2) the result of executing the template against router.Message is an empty string (probably not something anyone want). But it's it results in anything else, maybe that's what this person want.

**Comment by phobologic** [comment]
:+1:

**Pull request merged**